src/lib/dialog/dialog-config.ts

```diff
@@ -57,6 +57,18 @@ export class MatDialogConfig {
   /** Height of the dialog. */
   height?: string = '';
 
+  /** Min-width of the dialog. */
+  minWidth?: string;
+
+  /** Min-height of the dialog */
+  minHeight?: string;
+
+  /** Max-width of the dialog. */
```

Comment:
This comment should include `Defaults to 80vw`

Reply:
Done.